File: ios/ExpoRelativeTimeModule.swift
import ExpoModulesCore
import Foundation

public class ExpoRelativeTimeModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoRelativeTime")

    Function("getRelativeTime") { () -> Int in
        return Int(clock_gettime_nsec_np(CLOCK_MONOTONIC_RAW) / 1_000_000)
    }
  }
}

File: android/src/main/java/expo/modules/relativetime/ExpoRelativeTimeModule.kt
package expo.modules.relativetime

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition
import android.os.SystemClock

class ExpoRelativeTimeModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoRelativeTime")

    Function("getRelativeTime") {
      return@Function SystemClock.elapsedRealtime()
    }
  }
}
